addons/binding/org.openhab.binding.nibeuplink/src/main/java/org/openhab/binding/nibeuplink/internal/NibeUplinkHandlerFactory.java

```diff
@@ -0,0 +1,98 @@
+/**
+ * Copyright (c) 2010-2017 by the respective copyright holders.
+ *
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ */
+package org.openhab.binding.nibeuplink.internal;
+
+import static org.openhab.binding.nibeuplink.NibeUplinkBindingConstants.*;
+
+import org.eclipse.smarthome.core.thing.Thing;
+import org.eclipse.smarthome.core.thing.ThingTypeUID;
+import org.eclipse.smarthome.core.thing.binding.BaseThingHandlerFactory;
+import org.eclipse.smarthome.core.thing.binding.ThingHandler;
+import org.eclipse.smarthome.core.thing.binding.ThingHandlerFactory;
+import org.openhab.binding.nibeuplink.handler.ChannelSet;
+import org.openhab.binding.nibeuplink.handler.F1145Handler;
+import org.openhab.binding.nibeuplink.handler.F1155Handler;
+import org.openhab.binding.nibeuplink.handler.F750Handler;
+import org.openhab.binding.nibeuplink.handler.VVM310Handler;
+import org.openhab.binding.nibeuplink.handler.VVM320Handler;
+import org.osgi.service.component.annotations.Component;
+import org.osgi.service.component.annotations.ConfigurationPolicy;
+
+/**
+ * The {@link NibeUplinkHandlerFactory} is responsible for creating things and thing
+ * handlers.
+ *
+ * @author Alexander Friese - initial contribution
+ *
+ */
+@Component(service = ThingHandlerFactory.class, immediate = true, configurationPolicy = ConfigurationPolicy.OPTIONAL, name = "binding.nibeuplink")
```

Comment:
`configurationPolicy` can be omitted. The default value is `OPTIONAL`. `name` can also be omitted is will generate a xml with the classname. (I've seen bindings using the notation used here, but also some that just omitted then name).

Reply:
fixed it